src/WebGL.elm

```diff
@@ -232,5 +230,5 @@ Shaders and meshes are cached so that they do not get resent to the GPU,
 so it should be relatively cheap to create new entities out of existing values.
 -}
 toHtmlWith : Options -> List (Attribute msg) -> List Renderable -> Html msg
-toHtmlWith ({ settings } as options) =
-    Native.WebGL.toHtml options settings
+toHtmlWith =
+    Native.WebGL.toHtml
```

Comment:
🎨 when calling native functions, it is best to explictly list each argument when possible. That way it's 1) self documenting and 2) more obvious if something is wrong when you go from a FN to an FM